dice.py

```diff
@@ -0,0 +1,23 @@
+import random
+
+repeat = True
+while repeat:
```

Comment:
Rather than use a while loop it would probably be better to make this a function that is called recursively if the user enters Y at the end